### PR TITLE
Move to pdfkit

### PR DESCRIPTION
**Pull request by rbrtrfl**
- rename `/sections` folder with `/pdfForm` in `/domains/beratungshilfe/services/pdf`
- move related files into `/domains/prozesskostenhilfe/services/pdf/pdfForm`
- move `index.ts` from `/services/pdf/attachment to /domains/shared/pdf/attachment`
- move `/domains/shared/pdf` to `/domains/shared/services/pdf`
- move `/app/services/pdf/util` to `/app/domains/prozesskostenhilfe/services/pdf` along with corresponding tests
- move `/app/domains/share/services/pdf/attachment` to `/app/services/pdf/attachment`
- move `/app/domains/share/services/pdf/pdfStyles.ts` to `/app/services/pdf/`
- remove circular dependencies

### CODE REVIEW

File: app/domains/prozesskostenhilfe/services/pdf/__test__/util.test.ts [hunk 1]
```diff
@@ -45,4 +48,26 @@ describe("utility functions", () => {
       ).toBe("1.120,00");
     });
   });
+
+  describe("removeDecimalsFromCurrencyString", () => {
```
Comment (aaschlote) [inline app/domains/prozesskostenhilfe/services/pdf/__test__/util.test.ts:52]: this test is in the right folder?

File: app/domains/prozesskostenhilfe/services/pdf/util.ts [hunk 2]
```diff
@@ -24,3 +26,10 @@ export const getTotalMonthlyFinancialEntries = (
       minimumFractionDigits: 2,
       maximumFractionDigits: 2,
     });
+
+export function removeDecimalsFromCurrencyString(
```
Comment (aaschlote) [inline app/domains/prozesskostenhilfe/services/pdf/util.ts:30]: maybe it should be in the `app/util`, right?

File: app/domains/prozesskostenhilfe/services/pdf/__test__/util.test.ts (hunk 1 above)
Comment (rbrtrfl) [reply app/domains/prozesskostenhilfe/services/pdf/__test__/util.test.ts:52]: At the moment: Yes.

File: app/domains/prozesskostenhilfe/services/pdf/util.ts (hunk 2 above)
Comment (rbrtrfl) [reply app/domains/prozesskostenhilfe/services/pdf/util.ts:30]: The function is only used in the scope of filling pdf forms. So I would say it's correctly located where it is right now.
wdyt?

Comment (aaschlote) [reply app/domains/prozesskostenhilfe/services/pdf/util.ts:30]: it's only used in the scope of filling pdf forms, but the function itself doesn't have anything related the pdf in the source code, or am I missing something?

**Pull request merged**